conbench/entities/run.py

```diff
@@ -48,16 +48,17 @@ def create(data):
         )
         hardware = hardware_type.upsert(**data.pop(field_name))
 
-        sha, repository = None, None
+        sha, branch, repository = None, None, None
```

Comment:
I know this doesn't actually matter, but LOL that's a funny git diff that looks very wrong.

Reply:
🤣